vmdk_plugin/Makefile

```diff
@@ -114,6 +114,10 @@ build: prereqs .code_verify $(VM_BINS)
 gvt:
 	$(BUILD) gvt
 
+.PHONY: documentation
```

Comment:
nit: maybe add another target  ''docs' for those of us who like abbreviation :-)